data_scraper/data_scraper.py

```diff
@@ -1 +1,14 @@
-import asyncio
+import json
+from database.json_operations import insert_questions_data
+from config.storage_config import STORAGE_CONFIG
+
+
+async def scrap_data():
+    data = read_data_from_json_file()
+    await insert_questions_data(data)
```

Comment:
change send structure. It shoul be view like:
quiz_id=data[0],
topic=data[1], 
...